web-app/pages/index.tsx

```diff
@@ -1,11 +1,11 @@
 import React, { FormEvent, useState } from 'react';
 import type { NextPage } from 'next';
 import Layout from '../components/Layout';
-import styles from '../styles/Home.module.scss';
```

Comment:
@rashaabdulrazzak Can you please also delete the Home.module.scss file since I don't see usages of it anywhere in the project anymore?

Reply:
sure I deleted it